query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedConnectMutationInsideCreateSpec.scala

```diff
@@ -48,8 +48,8 @@ class NestedConnectMutationInsideCreateSpec extends FlatSpec with Matchers with
            |}
         """.stripMargin,
         project,
-        errorCode = 3042,
-        errorContains = "The change you are trying to make would violate the required relation 'ChildToParent' between Child and Parent"
+        errorCode = 0, // 3042,
+        errorContains = """Error in query graph construction: RelationViolation(RelationViolation { relation_name: \"ChildToParent\", model_a_name: \"Child\", model_b_name: \"Parent\""""
```

Comment:
Should `RelationViolation` be a user-facing error, with a code? (It sounds pretty common)

Reply:
yes